dot/state/storage.go

```diff
@@ -397,3 +397,8 @@ func (s *StorageState) pruneStorage(closeCh chan interface{}) {
 		}
 	}
 }
+
+// GetGenesisData retrieves current genesis data from database
+func (s *StorageState) GetGenesisData() (*genesis.Data, error) {
+	return LoadGenesisData(s.baseDB)
+}
```

Comment:
maybe we should have a separate `State` type for accessing things that aren't stored until the block, storage, etc. sub-databases, it would remove the need for baseDB. will open an issue